app/src/main/java/com/kamron/pogoiv/MainActivity.java

```diff
@@ -2,6 +2,7 @@
 
 import android.Manifest;
 import android.annotation.TargetApi;
```

Comment:
This is a bit tricky, I don't recommend it if you're not comfortable with `git`, nor should you apply the suggestions blindly.
If you want to learn the best approach for doing it, read on; that can be helpful or even essential when contributing to bigger projects. If that's too much, it's OK 😉 .

To split a commit into multiple ones, you can combine `git reset`, `git add -p` and `git commit` in the right sequence. For a commit at the base of a PR, you also need to do an interactive rebase with `git rebase -i`. To see how to combine those (in a very good explanation), the best source I've found is https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History#Changing-Multiple-Commit-Messages (there are answers on http://stackoverflow.com/q/6217156/53974 but they're less clear).

When you split a commit, each commit should not only make sense but at least still compile—so after making a commit, use `git stash` to "stash away" the uncommitted changes and try compiling the result. Trying to ensure that "Every commit compiles" helps when using `git bisect`.